docs/UserGuide.md

```diff
@@ -620,6 +735,83 @@ Important notes:
 
 #### Find Transaction(s)
 
+Finds a `Transaction` in UniCa$h.
+
+Command: `find n/NAME l/LOCATION c/CATEGORY`
```

Comment:
Could you use the standardized notation for optional parameters: `[n/Name]`, thanks!

Reply:
Ok sure